packages/components/psammead-brand/README.md

```diff
@@ -10,10 +10,12 @@ The `Brand` component provides the BBC service logo (as SVG), nested inside a st
 
 ## Props
 
+<!-- prettier-ignore -->
 | Argument  | Type   | Required | Default | Example      |
 | --------- | ------ | -------- | ------- | ------------ |
 | brandName | String | yes      | N/A     | `'BBC News'` |
 | svg | Object | yes | N/A | { group: `(<g fillrule="evenodd"><path d="M84.32" /></g>)`, viewbox: { height: 24, width: 167.95 }, ratio: 6.9979 } |
+| link | String | yes | N/A | `https://www.bbc.co.uk/news` |
```

Comment:
I personally would prefer this to be called `href`

Reply:
How about `url`